src/com/trilead/ssh2/packets/PacketKexInit.java

```diff
@@ -20,10 +20,10 @@
 
 	KexParameters kp = new KexParameters();
 
-	public PacketKexInit(CryptoWishList cwl, SecureRandom rnd)
+	public PacketKexInit(CryptoWishList cwl)
```

Comment:
Leave the old constructor signature and implementation in, and (if you do need the new constructor), create it as
```
public PacketKeyInit(CryptoWishList cw1) {
   this(cw1, new SecureRandom());
}
```